src/shared/helpers/translate.tsx

```diff
@@ -10,7 +10,9 @@ import i18n from '../translations/i18n';
  * @param key
  * @param params
  */
-export function tHtml(key: string, params?: TOptions | string | undefined): ReactNode | string {
+export function tHtml(key?: string, params?: TOptions | string | undefined): ReactNode | string {
+	if (!key) return;
```

Comment:
Does this ever happen? If so explicitly return undefined and type the response
```suggestion
export function tHtml(key?: string, params?: TOptions | string | undefined): ReactNode | string | undefined {
	if (!key) return undefined;
```